### PR TITLE
Feat: pod indicator reformat

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
# Pod indicator reformat

- Separate pod navigator into a component
- Style changes
- Tests updates

# Checklist

- [x] All acceptance criteria are met.
- [ ] Relevant documentation, if any, has been written/updated.
- [ ] The changelog has been updated, if applicable.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: components/containerPageHeader/styles.js [hunk 1]
```diff
@@ -22,17 +22,23 @@
...
         alignItems: "center",
+        paddingRight: 0,
+        borderBottom: `1px solid ${theme.palette.grey.A100}`,
       },
       display: "flex",
       flexDirection: "column",
       alignItems: "flex-start",
+      paddingLeft: 0,
+      "& h1": {
+        paddingLeft: "24px",
```
Comment (megoth) [inline components/containerPageHeader/styles.js:40]: Use the theme enabled spacing instead, e.g. `theme.spacing(2.4)` (I think default spacing is 10).

File: components/podIndicator/podNavigatorPopover/styles.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { createStyles } from "@solid/lit-prism-patterns";
+
+const styles = (theme, popoverWidth) =>
+  createStyles(theme, [], {
+    popoverNavigator: {
+      borderRadius: "0 0 10px 10px",
```
Comment (megoth) [inline components/podIndicator/podNavigatorPopover/styles.js:27]: Again, let's use the theme's spacing method: `theme.spacing(0, 0, 1, 1)`.

File: components/podIndicator/styles.js [hunk 3]
```diff
@@ -21,36 +21,94 @@
...
+      fontWeight: theme.typography.fontWeightBold,
+      color: theme.palette.primary.main,
+      textTransform: "uppercase",
+    },
+    indicatorChevron: {
+      paddingLeft: "0.5em",
+      color: theme.palette.primary.main,
     },
     indicatorPrompt: {
+      padding: "6px 24px",
```
Comment (megoth) [inline components/podIndicator/styles.js:51]: Use theme.spacing.

File: components/podIndicator/styles.js [hunk 4]
```diff
@@ -21,36 +21,94 @@
...
+        maxWidth: "50vw",
+        minWidth: "128px",
+        width: "max-content",
+        height: "60px",
+      },
     },
     indicatorLabel: {
-      fontSize: "0.825rem",
-      fontWeight: theme.typography.fontWeightMedium,
+      fontSize: "13px",
```
Comment (megoth) [inline components/podIndicator/styles.js:41]: I think we should use 0.825rem here, to stay consistent with other uses of small fonts.

File: components/podIndicator/styles.js [hunk 5]
```diff
@@ -21,36 +21,94 @@
...
+      borderTop: `1px solid ${theme.palette.grey.A100}`,
+      borderBottom: `1px solid ${theme.palette.grey.A100}`,
       textTransform: "none",
       fontSize: "0.825rem",
       fontWeight: theme.typography.fontWeightRegular,
+      [theme.breakpoints.up("sm")]: {
+        border: "none",
+        borderLeft: `1px solid ${theme.palette.grey.A100}`,
+        alignItems: "flex-end",
+        padding: `6px 40px 3px ${indicatorLabelPaddingLeft}px`,
```
Comment (megoth) [inline components/podIndicator/styles.js:71]: Use theme.spacing

File: components/podIndicator/styles.js [hunk 6]
```diff
@@ -21,36 +21,94 @@
...
+        textOverflow: "ellipsis",
+        overflow: "hidden",
+        whiteSpace: "nowrap",
       },
     },
+    popoverMenu: {
+      maxWidth: `max(${indicatorWidth}px, 170px)`,
+      minWidth: "128px",
+      width: `max(${indicatorWidth}px, 170px)`,
+      borderRadius: "0 0 10px 10px",
```
Comment (megoth) [inline components/podIndicator/styles.js:90]: Use theme.spacing

File: components/podIndicator/styles.js [hunk 7]
```diff
@@ -21,36 +21,94 @@
...
+      },
+    },
+    list: {
+      padding: 0,
+    },
+    menuItem: {
+      padding: theme.spacing(1),
+    },
+    itemIcon: {
+      padding: "0 8px",
```
Comment (megoth) [inline components/podIndicator/styles.js:105]: Use theme.spacing

**Review by megoth — CHANGES_REQUESTED** [review]
We should translate px to the corresponding values in theme.spacing.

File: components/podIndicator/styles.js [hunk 8]
```diff
@@ -38,7 +38,7 @@ const styles = (theme, indicatorWidth, indicatorLabelWidth) => {
       },
     },
     indicatorLabel: {
-      fontSize: "13px",
+      fontSize: theme.typography.subtitle2,
```
Comment (megoth) [inline components/podIndicator/styles.js:41]: You should be more specific, i.e. `theme.typography.subtitle2.fontSize`.

**Review by megoth — APPROVED** [review]
Just another slight change, and we're good; I'll approve it at once, so you can just mark it ready for QA when you're done.

**Review by megoth — APPROVED** [review]

**Review by kevin-inrupt — APPROVED** [review]

**Pull request merged**